### ISSUE TITLE
NVDA doesn't announce radio button's status after I press "Space"

### ISSUE THREAD

**Issue by kceiw** [nvaccess/nvda#8058]
### Steps to reproduce:

> 1. Run the attach WPF application. It displays three radio buttons
[WpfApp1.zip](https://github.com/nvaccess/nvda/files/1782148/WpfApp1.zip)
> 2. Use up/down arrows key to navigate to any unselected radio button
> 3. Press "Space"
> 4. Notice that NVDA doesn't announce anything.

### Expected behavior:
> NVDA announces the status of the radio button for example: "Option 2 radio button checked"

### Actual behavior:
> NVDA doesn't announce anything. In the case that it announce, it says "Space".

### System configuration:
NVDA version:
> 2017.4

NVDA Installed or portable:
> NVDA is installed

Windows version:
> Windows 10 Version 1709 Build 16299

Name and version of other software in use when reproducing the issue:

### Other questions:

Does the issue still occur after restarting your PC?
> Yes

Have you tried any other versions of NVDA?
> No

**Comment by ehollig** [comment]
I think we may need some more information. For example, how was this example program made?

**Comment by kceiw** [comment]
I tried NVDA 2018.3.2. It doesn't announce "Space" right now. It announce nothing.

The way to create this program is:
1. Create a WPF application in Visual Studio using the template WPF App (.Net Framework).
2. Add a few RadioButton controls.

Then you can build the project and run the application.

**Comment by Adriani90** [comment]
Actually this is expected behavior. Radio buttons do not allow for multi selection, so any interface that uses radio buttons activates the coresponding radio button when the keyboard system focus lands on it. In this case, the expected behavior should be that the radio button is activated when I press up and down arrow to switch between radio buttons.
In these cases spacebar is not needed.
This is how it works in every form with radio buttons, also in browsers. NVDA announces "radio button enabled" when landing with the focus on a radio button using arrow keys.
This is the difference to checkboxes which allow for multi selection, that's why we use spacebar to check or uncheck them.

I am closing this request as invalid.

**Comment by kfam98** [comment]
Also, an issue with our current controls using a radio button in VS. We support only single selection, but not necessarily selecting the button with keyboard focus, resulting in the selection not being announced.
